content/en/docs/distributions/charmed/install-kubeflow.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Installing Kubeflow"
+description = "Instructions for Kubeflow deployment with Kubeflow Charmed Operators"
+weight = 10
++++
+
+This guide lists the steps necessary to install Kubeflow on any conformant Kubernetes, including AKS, EKS, GKE, Openshift and any kubeadm-deployed cluster,  provided that you have access to it via `kubectl`. 
```

Comment:
Suggestion: 

- Maybe make it a bit more "active"/engaging wrt users
- Spell out GKE, EKS, AKS, GKE and provide links to their docs to ease UX friction
- "OpenShift"
- Add a link to kubeadm (for less experienced users)

e.g.

```suggestion
This guide will go through the steps you need to install Kubeflow on any conformant Kubernetes, including [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/), [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/index.html), [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/docs/), [OpenShift](https://docs.openshift.com), and any [kubeadm](https://kubernetes.io/docs/reference/setup-tools/kubeadm/)-deployed cluster (provided that you have access to it via `kubectl`). 
```

LMKWYT @RFMVasconcelos  👍

Reply:
This looks good!

Reply:
Thank you!!